### ISSUE TITLE
Add renovate to repositories

### ISSUE THREAD

**Issue by marklise** [bcgov/parks-data-register#442]
https://github.com/apps/renovate

https://github.com/bcgov/quickstart-openshift/pull/1898

**Comment by marklise** [comment]
@JLWade @Dianadec coming from NRIDS, for refinement.

**Comment by JLWade** [comment]
is this the same as: https://github.com/bcgov/parks-data-register/issues/372

**Comment by LindsayMacfarlane** [comment]
@marklise - do we know if this one has been done and/or is still needed?

**Comment by marklise** [comment]
Not done - and we should do it @LindsayMacfarlane

**Comment by LindsayMacfarlane** [comment]
@Dianadec - thinking we should look at breaking this down to each repository? @marklise thoughts on that approach?

**Comment by Dianadec** [comment]
Yeah that would make sense to me. This is a duplicate of #372 so we could use this as one of the apps if marks confirm that would make sense. Would it be easier to lump or split from a dev perspective?
